src/util/symbol_generator.h

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Could you add  a header like: https://github.com/esbmc/esbmc/blob/master/src/util/algorithms.h

Reply:
Strange to have copyright/authorship information only in the interface definition (header file). Who wrote the actual code I just moved to this separate class in order to not duplicate it, I don't know.

Reply:
I'm ok with skipping authorship. But it is still good to know why you created a new header file, it makes easier to use/extend later.

Reply:
done